cluster/emulation.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: allow-emulation
```

Comment:
Not sure if this was addressed before, but: What about having one ConfigMap called `kubevirt-config`, which then contains `debug.allowEmulation: true`, i.e.:

```yaml
kind: ConfigMap
metadata:
  name: kubevirt-config
  namespace: kube-system
data:
  debug.allowEmulation: true
```

Reply:
That was actually what we agreed on through the design phase. @stu-gott why did you decide to use an `emulation` focused config map?

Reply:
The file `cluster/emulation.yaml` was from an early iteration of the code and should have been deleted. Sorry for the confusion.